test/test-estatjp.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'pathname'
+
+class EstatjpTest < Test::Unit::TestCase
+  sub_test_case('test') do
```

Comment:
Can we remove this `sub_test_case` because we have only one `sub_test_case`.

Reply:
I removed this unnecessary `sub_test_case`.